force-app/main/default/reports/NPSP_Outbound_Fund_Reports/Aging_Disbursements.report-meta.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Report xmlns="http://soap.sforce.com/2006/04/metadata">
+    <chart>
+        <backgroundColor1>#FFFFFF</backgroundColor1>
+        <backgroundColor2>#FFFFFF</backgroundColor2>
+        <backgroundFadeDir>Diagonal</backgroundFadeDir>
+        <chartSummaries>
+            <aggregate>Sum</aggregate>
+            <axisBinding>y</axisBinding>
+            <column>Funding_Program__c.FundingRequests__r.Disbursements__r$Amount__c</column>
+        </chartSummaries>
+        <chartType>VerticalColumnStacked</chartType>
+        <enableHoverLabels>false</enableHoverLabels>
+        <expandOthers>true</expandOthers>
+        <groupingColumn>CDF1</groupingColumn>
+        <legendPosition>Right</legendPosition>
+        <location>CHART_BOTTOM</location>
+        <secondaryGroupingColumn>Funding_Program__c$Name</secondaryGroupingColumn>
+        <showAxisLabels>true</showAxisLabels>
+        <showPercentage>false</showPercentage>
+        <showTotal>false</showTotal>
+        <showValues>false</showValues>
+        <size>Medium</size>
+        <summaryAxisRange>Auto</summaryAxisRange>
+        <textColor>#000000</textColor>
+        <textSize>12</textSize>
+        <title>Overdue Disbursements</title>
+        <titleColor>#000000</titleColor>
+        <titleSize>18</titleSize>
+    </chart>
+    <columns>
+        <field>Funding_Program__c.FundingRequests__r.Disbursements__r$Funding_Request__c</field>
+    </columns>
+    <columns>
+        <field>Funding_Program__c.FundingRequests__r.Disbursements__r$Name</field>
+    </columns>
+    <columns>
+        <field>Funding_Program__c.FundingRequests__r.Disbursements__r$Scheduled_Date__c</field>
+    </columns>
+    <columns>
+        <aggregateTypes>Sum</aggregateTypes>
+        <field>Funding_Program__c.FundingRequests__r.Disbursements__r$Amount__c</field>
+    </columns>
+    <columns>
+        <field>Funding_Program__c.FundingRequests__r.Disbursements__r$Status__c</field>
+    </columns>
+    <description>List of overdue disbursements group by funding program.</description>
```

Comment:
@nicolersmith - I added `List of overdue disbursements group by funding program`  as the description of the report.